spacy_experimental/coref/span_resolver_component.py

```diff
@@ -0,0 +1,403 @@
+from typing import Iterable, Tuple, Optional, Dict, Callable, Any, List
+import warnings
+
+from thinc.types import Floats2d, Floats3d, Ints2d
+from thinc.api import Model, Config, Optimizer, CategoricalCrossentropy
+from thinc.api import set_dropout_rate, to_categorical
+from itertools import islice
+import srsly
+
+from spacy.pipeline.trainable_pipe import TrainablePipe
+from spacy.language import Language
+from spacy.training import Example, validate_examples, validate_get_examples
+from spacy.errors import Errors
+from spacy.tokens import Doc
+from spacy.vocab import Vocab
+from spacy.util import from_bytes, from_disk
+
+from .coref_util import (
+    MentionClusters,
+    DEFAULT_CLUSTER_PREFIX,
+)
+
+from .coref_scorer import doc2clusters, score_span_predictions
+
+default_span_resolver_config = """
+[model]
+@architectures = "spacy-experimental.SpanResolver.v1"
+hidden_size = 1024
+distance_embedding_size = 64
+conv_channels = 4
+window_size = 1
+max_distance = 128
+prefix = "coref_head_clusters"
+
+[model.tok2vec]
+@architectures = "spacy.Tok2Vec.v2"
+
+[model.tok2vec.embed]
+@architectures = "spacy.MultiHashEmbed.v1"
+width = 64
+rows = [2000, 2000, 1000, 1000, 1000, 1000]
+attrs = ["ORTH", "LOWER", "PREFIX", "SUFFIX", "SHAPE", "ID"]
```

Comment:
Same suggestion as above.

```suggestion
attrs = ["NORM", "LOWER", "PREFIX", "SUFFIX", "SHAPE"]
rows = [5000, 2500, 1000, 2500, 2500]
```